frontend/components/PublishButton.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+import Icon from "react-native-vector-icons/Ionicons";
+import { useNavigation } from "expo-router";
+import axios from "axios";
+
+interface ReviewCardProps {
+  rating: number;
+  review: string;
+}
+
+const PublishButton: React.FC<ReviewCardProps> = ({ rating, review }) => {
+  const handleClick = () => {
+    console.log("Publishing review");
+    axios
+      .post(`http://10.110.235.22:8080/reviews`, {
```

Comment:
don't forget to swap in the real value here 

`process.env.EXPO_PUBLIC_BASE_URL`